### ISSUE TITLE
block context should consider child blocks

### ISSUE THREAD

**Issue by Ingegneus** [omagdy7/ollama-logseq#30]
what the titel says. I think it makes much more sense, since logseq puts so much emphasis on atomic info in separate bocks

**Comment by Glitch-guy0** [comment]
yeah it doesn't consider child blocks. I mean i love this tool, but don't know how to use this 🫤

**Comment by omagdy7** [comment]
This is how it works I just tried it on a some block with sub sub bullet points and it worked can you give me something where you felt the context of child block weren't considered?

**Comment by Glitch-guy0** [comment]
https://github.com/user-attachments/assets/7efb8ffd-6225-4ca0-9b7b-89256eb15fbe

hey @omagdy7, it's not working for me, using gemma:7b, and i have a iGPU just in case : )
